it/spring/boot2-mixed-tomcat9/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    implementation project(':spring:boot2-starter')
+    implementation project(':spring:boot2-webflux-starter')
+    implementation project(':tomcat9')
+    implementation('org.springframework.boot:spring-boot-starter-web') {
+        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
```

Comment:
I refer to [it/spring/boot2-tomcat9/build.gradle](https://github.com/line/armeria/blob/master/it/spring/boot2-tomcat9/build.gradle).

Since `spring-boot-starter-tomcat` only has dependencies, I think this is simply a `tomcat-embed-core` version difference. The behavior will be the same.